contribs/gmf/less/mobile.less

```diff
@@ -118,7 +118,7 @@ span.twitter-typeahead {
     position: absolute;
     top: 0;
     left: 50%;
-    margin-left: ~"calc("@search-input-size / 2 ~" - " @map-tools-size~")";;
+    margin-left: ~"calc("@search-input-size-clear / 2 ~" - " @map-tools-size~")";
```

Comment:
The `~"..." ` is used to say to "less" to not interpret this content. That means that the browser will do the calculation. When we can, we prefer that less do the calculation, so our application is a little bit faster.

In this case, all units are in px (the "2" can be in pixel or other thing, that doesn't change the result) so less can do the complete job and I pretty sur that we can write this line like this:
`margin-left: calc(@search-input-size-clear / 2 - @map-tools-size);`

That's easier to read and faster for the browser.